File: Xamarin/CommandCenter.Xamarain/CommandCenter.Xamarain/Models/Sensor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CommandCenter.Xamarain.Models
{
    public class Sensor
    {
        public string Type { get; set; }
        public string Location { get; set; }
        public string Value { get; set; }
        public string Service { get; set; }
        public int Position { get; set; }
    }
}

File: Android/app/src/main/java/com/pop/commandcenter/models/Sensor.java
package com.pop.commandcenter.models;

/**
 * Created by joemc on 12/26/2017.
 */

public class Sensor {
    public String getType() {
        return Type;
    }

    public void setType(String type) {
        Type = type;
    }

    public String getLocation() {
        return Location;
    }

    public void setLocation(String location) {
        Location = location;
    }

    public String getValue() {
        return Value;
    }

    public void setValue(String value) {
        Value = value;
    }

    public String getService() {
        return Service;
    }

    public void setService(String service) {
        Service = service;
    }

    public int getPosition() {
        return Position;
    }

    public void setPosition(int position) {
        Position = position;
    }

    public String Type;
    public String Location;
    public String Value;
    public String Service;
    public int Position;
}
